src/components/Main.jsx

```diff
@@ -7,9 +7,10 @@ import UploadSnippet from './UploadSnippet';
 import About from './About';
 import SignIn from './SignIn';
 
+import '../styles/main/main.styl';
 
 const Main = () => (
-  <main>
+  <main className="main">
     <Switch>
       <Route exact path="/" component={NewSnippet} />
       <Route path="/recent" component={RecentSnippets} />
```

Comment:
Hm.. so the route are still there but you removed any navigation links to them. Why?

Reply:
I didn't remove them, this file has main routing but NavLinks is a way to navigate through them from different components. So far almost all NavLinks (except SignIn) are on Sidebar component